### PR TITLE
Add mysqlworkbench version 6.3.10 for High Sierra users

### PR DESCRIPTION
**Pull request by ottnorml**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/mysqlworkbench.rb [hunk 1]
```diff
@@ -1,13 +1,18 @@
 cask 'mysqlworkbench' do
-  version '8.0.18'
-  sha256 '965f85163d1723be26c1f0c74c5b1cd908fac79e02c00fa371c217c9a3bf09ae'
+  if MacOS.version <= :high_sierra
+    version '6.3.10'
+    sha256 :no_check
```
Comment (ottnorml) [inline Casks/mysqlworkbench.rb]: The checksum should be `29857bf84bebb7c4442ce147e44602d00f8c001e3c09b3a6e3af356767e08d2c`, but `brew cask style --fix` replaced it. How can I fix that?

Comment (vitorgalvao) [inline Casks/mysqlworkbench.rb]: This still needs to be set.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/mysqlworkbench.rb (hunk 1 above)
Comment (ottnorml) [reply Casks/mysqlworkbench.rb]: I added the checksum in 6836936.

**Comment by vitorgalvao** [comment]
Looking good. Thank you.

**Review by vitorgalvao — APPROVED** [review]

**Comment by ottnorml** [comment]
@vitorgalvao Do you have an idea, why the `xcode10.1` Travis build ([143135.1](https://travis-ci.com/Homebrew/homebrew-cask/jobs/245980573)) is failing?

**Comment by reitermarkus** [comment]
The `appcast` doesn't apply for the High Sierra version.

**Comment by ottnorml** [comment]
@reitermarkus Is this a problem? `find_appcast` did not find anything and I did not change it.

**Comment by vitorgalvao** [comment]
> The `appcast` doesn't apply for the High Sierra version.

Technically it does, because it links back to the `6.3` versions. But presumably version 6 won’t get further updates, so it shouldn’t be a problem (if it is, we can just stick the `appcast` in the conditional).

**Comment by John-Dennert** [comment]
Per https://dev.mysql.com/downloads/workbench/, MySQL Workbench 8.0.18 works with High Sierra. "Packages for Mojave (10.14) are compatible with High Sierra (10.13)"

**Comment by ottnorml** [comment]
> Per https://dev.mysql.com/downloads/workbench/, MySQL Workbench 8.0.18 works with High Sierra. "Packages for Mojave (10.14) are compatible with High Sierra (10.13)"
In this case, it must be new in version 8.0.18, because the program in version 8.0.17 with High Sierra refused to run (see #69926).

**Comment by ottnorml** [comment]
Screenshot
As you can see from the screenshot, version `8.0.18` is not compatible with High Sierra.

**Comment by ottnorml** [comment]
@vitorgalvao  & @reitermarkus How do we continue?

**Pull request merged**